corehq/apps/app_manager/static/app_manager/js/case-config-ui-advanced.js

```diff
@@ -402,7 +402,7 @@ hqDefine('app_manager/js/case-config-ui-advanced.js', function () {
                         fixture_tag: '',
                         fixture_variable: '',
                         case_property: '',
-                        auto_select: false,
+                        auto_select: (COMMCAREHQ.toggleEnabled('APP_MANAGER_V2')) ? '' : false,
```

Comment:
@biyeun looks like this line from this PR 7 months ago is causing https://manage.dimagi.com/default.asp?257934. Essentially, if you add a "Load Case From Fixture" Action to an Advanced Form, and then save immediately, the server 500s because it can't wrap `auto_select` value '' since it's expecting a boolean.

If I just undo this line and change it back to `false` it seems like everything works. Any idea if I'm missing something in the intention behind this line?